server/__test__/api/search.js

```diff
@@ -0,0 +1,29 @@
+const request = require("supertest");
+const mongoose = require("mongoose");
+const buildDB = require("./../../database/dummyData/index");
+const app = require("./../../app");
+
+describe("testing search route", () => {
+  beforeAll(async () => {
+    await buildDB();
+  });
+  afterAll(async () => {
+    await mongoose.disconnect();
+  });
+  beforeEach(async () => {
+    await buildDB();
+  });
+
+  test("test valid request", (done) => {
```

Comment:
it would be good to test that you get some specific data back when you search for something